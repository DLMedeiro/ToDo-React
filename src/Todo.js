import React from "react";

const Todo = ({ id, todo, remove }) => {
  const handleRemove = () => {
    remove(id);
  };

  return (
    <div>
      <li id={id}>
        {todo}
        <button onClick={handleRemove}>Remove Activity</button>
      </li>
    </div>
  );
};

export default Todo;
