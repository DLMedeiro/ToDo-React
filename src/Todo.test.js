import React from "react";
import { render } from "@testing-library/react";
import Todo from "./Todo";

test("it renders without crashing", () => {
  render(<Todo />);
});

it("matches snapshot", function () {
  const { asFragment } = render(<Todo />);
  expect(asFragment()).toMatchSnapshot();
});
