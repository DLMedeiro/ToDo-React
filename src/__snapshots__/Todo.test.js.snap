// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <li>
      <button>
        Remove Todo
      </button>
    </li>
  </div>
</DocumentFragment>
`;
